src/main/java/repo/Order/OrderRepo.java

```
@@ -0,0 +1,36 @@
+package repo.Order;
+
+
+import java.util.HashMap;
+import java.util.Map;
+
+
+public class OrderRepo {
+
+    private Map<Integer, Order> mapOrders = new HashMap<>(1);
+
+
+    public void putOrderInMap(Integer id, Order order) {
+        mapOrders.put(id, order);
+
+    }
+
+    public Order getOrderById(Integer id) {
+        if (mapOrders.get(id) == null) throw new RuntimeException("Cant order a non present product");
```

Fail fast, top! :heavy_check_mark: 
Als gute Angewohnheit würde ich empfehlen stets `{}` beim if zu benutzen (sicher ist sicher, irgendwann willst du mal ne Zeile hinzufügen und merkst vlt nicht, dass diese nicht vom if-Block übernommen wird, weil ohne geschweifte Klammern nur die erste if Zeile gilt)